agents/components/MemberInvites.js

```diff
@@ -5,76 +5,95 @@ import { connect as connectFela } from 'react-fela'
 import { Field, FieldArray, formValueSelector, reduxForm as connectForm } from 'redux-form'
 import { flow } from 'lodash'
 import { TextField } from 'redux-form-material-ui'
+import MuiTextField from 'material-ui/TextField'
```

Comment:
the "next row" is not yet part of the redux form state (as part of this alternative approach), so i felt it was best to represent it outside of the redux form components. i did try this as a redux form `<Field />`, but it felt weird to guess what the required `name` prop would be (`members[${fields.length}]`).